tests/backdrop/UserCommandsTest.php

```diff
@@ -35,9 +35,9 @@ public function test_user_password_command_works() {
    * Make sure that the user-login command works.
    */
   public function test_user_login_command_works() {
-    $output = shell_exec('bee user-login admin');
+    $output = shell_exec('bee --base-url=https://bee.lndo.site user-login admin');
     $this->assertStringContainsString("Use the following link to login as 'admin':", (string) $output);
-    $this->assertStringContainsString('/user/reset/1/', (string) $output);
+    $this->assertStringContainsString('https://bee.lndo.site/user/reset/1/', (string) $output);
```

Comment:
I don't think this test should use this global option. It should work without it. If we want to test the global option, we should add a new test for it, ideally in `BeeCoreTest.php` with the tests for the other global options.

Reply:
Done